tests/cupy_tests/core_tests/test_ndarray_ufunc.py

```diff
@@ -88,7 +88,7 @@ def test_indexing(self, xp):
         b = cupy.testing.shaped_arange((3, 2), xp)[:, None, :]
         return a * b
 
-    @testing.with_requires('numpy>=1.11')
+    @testing.with_requires('numpy>=1.13')
```

Comment:
See https://docs.scipy.org/doc/numpy-1.13.0/release.html#ufunc-behavior-for-overlapping-inputs.